modules/migration-configuring-mcg.adoc

```diff
@@ -25,16 +25,14 @@ ifdef::openshift-origin[]
 +
 If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
 endif::[]
-* You must deploy {rh-storage} by using the appropriate link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9[OpenShift Data Foundation deployment guide].
+* You must deploy {rh-storage} by using the appropriate link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12[OpenShift Data Foundation deployment guide].
 
 .Procedure
 
-. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY` by running the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the `NooBaa` custom resource.
-ifdef::installing-3-4,installing-mtc[]
-+
-You use these credentials to add MCG as a replication repository.
-endif::[]
-ifdef::installing-oadp-mcg[]
+. Create a bucket using MCG resources according to link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html-single/managing_hybrid_and_multicloud_resources/index#doc-wrapper[Managing hybrid and multicloud resources guide].
+
+. Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `bucket name` by running the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] to the bucket resource created.
```

Comment:
@bergerhoffer For the inks present here, for example, "You must deploy {rh-storage} by using the appropriate link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12[OpenShift Data Foundation deployment guide].", how to write it as per the guidelines? Especially, according to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections.

Reply:
That one is okay to keep as is. Since we are not pointing to our own OCP docs, you're pointing to another product's docs (ODF). That only exists as a book on the portal, and we can't xref in that case, so keeping a hard link to the book on the portal is fine!

Reply:
> The first part of this sentence is a bit confusing to me. Is the S3 endpoint different than `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `bucket name`, or are those the 3 things that make up the S3 endpoint.
> 
> Also it looks odd to me that "bucket name" is in backticks. Is that something we're literally looking for in the output of the describe command in that guide? (If so, I don't see it). Probably shouldn't be in backticks (unless it gets changed to an actual field name to look for), and maybe needs cleared up some other way.
> 
> I also don't see how these items fit into the command below (other than the two `AWS_*` ones), so clearing that up some might help

Reply from @mattmuench:-

"The S3 endpoint (URL), "login name" =AWS_ACCESS_KEY_ID  and "password"=AWS_SECRET_ACCESS_KEY are the credentials communicate with the S3 endpoint for the bucket bucket name .
bucket_name should not be in backticks (see the Bucket Name: line under Spec:):
Example from my system:
[root@ocp62-jump ~]# oc describe -n openshift-adp obc/oadp-mcg
....
Spec:
 Bucket Name:      oadp-mcg-s3-bucket-40ad25b9-954a-4fdf-80f8-d5a0dc97ba99
 Generate Bucket Name: oadp-mcg-s3-bucket
 Object Bucket Name:  obc-openshift-adp-oadp-mcg
 Storage Class Name:  [openshift-storage.noobaa.io](http://openshift-storage.noobaa.io/)
....
[root@ocp62-jump ~]# oc get -n openshift-storage route/s3
NAME  HOST/PORT                   PATH  SERVICES  PORT    TERMINATION    WILDCARD
s3   s3-openshift-storage.apps.ocp62.dslab.local     s3     s3-https  reencrypt/Allow  None
[root@ocp62-jump ~]#

=> the latter gives you the endpoint (assembled): https://s3-openshift-storage.apps.ocp62.dslab.local/"

Reply:
I checked the discussion. It seems to me that this was all about it. Do we need something in addition ?

Reply:
@bergerhoffer ptal

Reply:
Hi @subhtk, sorry for missing your tag. Thanks @mattmuench for clarifying. 

So it sounds like it all will make sense to a user following the procedure, so that's fine to keep as is. So then the only thing I think could be changed is `bucket name` being in backticks. Per matt here:

> bucket_name should not be in backticks (see the Bucket Name: line under Spec:):

Or perhaps you could keep it in backticks, but then it should be capitalized as `Bucket Name` as the `oc describe` command will actually show. Up to you!

Reply:
I would suggest to go with the Bucket Name in back ticks. It's referring only to MCG and thus we could leverage exactly what a user should look for. 
Instead, I would go with bucket_name only if this would be generic and not really about MCG and not about OBC. But this is not what we want to document here.

Reply:
@bergerhoffer Is it resolved now?

Reply:
You'd need to capitalize it to `Bucket Name`, then I think we'd be good!